homeassistant/components/repetier/sensor.py

```diff
@@ -180,12 +180,16 @@ def update(self):
             return
         job_name = data['job_name']
         start = data['start']
-        print_time = data['print_time']
+        end = data['print_time']
         from_start = data['from_start']
-        time_end = start + round(print_time, 0)
-        self._state = datetime.utcfromtimestamp(time_end).isoformat()
-        remaining = print_time - from_start
+        time_end = start + round(end, 0)
+        remaining = end - from_start
         remaining_secs = int(round(remaining, 0))
+        state = time.strftime('%H:%M:%S', time.gmtime(remaining_secs))
+        self._state = state
+        end_time = datetime.utcfromtimestamp(time_end).isoformat()
+        self._attributes['finish'] = end_time
+        self._attributes['seconds'] = remaining_secs
```

Comment:
Relative time isn't allowed in state attributes. It will spam the state machine.

Reply:
The API only returns one none relative time, start. Everything else is a relative calculation from start.

Reply:
If relative times are not allowed, then we'll need to remove elapsed and remaining sensors and perhaps add these times as attributes to the jobsensor.